frappe/desk/reportview.py

```diff
@@ -31,7 +31,9 @@ def get_list():
 @frappe.read_only()
 def get_count():
 	args = get_form_params()
-	args.fields = ['{distinct}count(name) as total_count'.format(distinct = 'distinct ' if args.distinct=='true' else '')]
+
+	distinct = 'distinct ' if args.distinct=='true' else ''
+	args.fields = [f"count({distinct}`tab{args.doctype}`.name) as total_count"]
```

Comment:
I don't know if we support f-strings yet.

Reply:
Sorry, I re-checked. Frappe uses a lot of f-strings. I thought we still supported python 3.5 :sweat_smile:

Reply:
Yes, we are using them a lot. But I am not sure this is a conscious decision taken or they just seeped in.